resources/default-settings/jukebox.default.yaml

```diff
@@ -87,6 +87,10 @@ playermpd:
     update_on_startup: true
     check_user_rights: true
   mpd_conf: ~/.config/mpd/mpd.conf
+  play_position_tracker:
+    resume_by_default: false
+    file: ../../shared/settings/play_positions.json
```

Comment:
Just a remark: it's not a setting from my point of view, so maybe `shared/logs` is a better place?

Reply:
Changed.